packages/driver/src/multi-domain/communicator.ts

```diff
@@ -144,15 +101,26 @@ export class SpecBridgeDomainCommunicator extends EventEmitter {
   private windowReference
 
   private handleSubjectAndErr = (data: any = {}, send: (data: any) => void) => {
-    const { subject, err, ...rest } = data
+    let { subject, err, ...rest } = data
+
+    // check to see if the 'err' key is defined, and if it is, we have an error of any type
+    const hasMultiDomainError = !!Object.getOwnPropertyDescriptor(data, 'err')
```

Comment:
In what case does simply checking for `err` not work?

Reply:
when `err` is `undefined` or falsy. [This](https://github.com/cypress-io/cypress/pull/20520/files#diff-24a3601e6f74e7b820b3d189b306a63af9acc391217e5e0318aff8be058279daR451) test case fails and hangs the test forever because it doesn't think an error has occured. Super rare, but does cover the falsy edge case.

Reply:
I guess it just rubs me the wrong way to need this extra property for a couple rare edge cases when otherwise it's self-evident. How about we wrap the `err` in that case? Something like:

```js
// give the `err` truthiness if it's a falsey value like undefined/null/false
if (hasMultiDomainError && !err) {
  err = new Error(err)
}
```

Reply:
I think as long as we cast the `err` to a string in the constructor, the behavior should be the same. Will update shortly!

Reply:
addressed in f6c6975bfe